schedule, budget,risk profile, and approval sections edited (#9)
Capstone-Skynet/Capstone-Skynet.github.io: docs/proposal/3_8_risk_profile.tex

\endfoot
\hline
\endlastfoot

> Muchen become hostile&1&1&1\\ \hline

Reviewer: ```suggestion
```

Electronics and battery are too heavy for the drone to take off.&0.5&1&0.5\\ \hline
Drone cannot take off at all due to drone hardware failure.&0.5&1&0.5\\ \hline
Accidents that damage to drone and FPGA that leads to extra budget that we may not have.&0.7&0.7&0.49\\ \hline
Forced to buy cheaper components which leads to lower performance.&0.8&0.5&0.4\\ \hline
Deliveries is not meeting client’s expectations&0.2&0.8&0.4\\ \hline
Knowledge and skill (Machine Learning and Computer vision) required for project is not sufficient.&0.5&0.8&0.4\\ \hline
Our scrum method is not efficient&0.5&0.7&0.35\\ \hline
Scope of the project is underestimated which leads to burn outs&0.5&0.7&0.35\\ \hline
Not have enough money.&0.5&0.7&0.35\\ \hline
Not enough time commitment from team member due to other courses.&0.5&0.6&0.3\\ \hline
Team is indecisive which leads to delay - management is not good enough to make a timely decision.&0.4&0.7&0.28\\ \hline
Team has lack of communication and make assumptions which leads to incompatibility&0.4&0.7&0.28\\ \hline
Documentation is not specific / complete.&0.7&0.4&0.28\\ \hline
Not enough time to work on documentation.&0.7&0.4&0.28\\ \hline

> Peter become hostile&0.5&0.5&0.25\\ \hline

Reviewer: ```suggestion
```

Scope of the project is overestimated which leads to diversification of vision of the project.&0.5&0.5&0.25\\ \hline
Not enough machine learning training data.&0.5&0.5&0.25\\ \hline

> Arthur becomes hostile&0.5&0.5&0.25\\ \hline

Reviewer: ```suggestion
```

> Wilson becomes hostile&0.5&0.5&0.25\\ \hline

Reviewer: ```suggestion
```

Illegal to fly drone.&0.3&0.8&0.24\\ \hline
New technology or research that changes the scope significantly&0.3&0.7&0.21\\ \hline
Technical debt paydown impact project timeline.&0.4&0.5&0.2\\ \hline
Software license does not allow our application to be delivered.&0.3&0.6&0.18\\ \hline
Key hardware components not available&0.2&0.8&0.16\\ \hline
Machine learning resources lack documentation&0.3&0.5&0.15\\ \hline
Camera lack documentation&0.3&0.5&0.15\\ \hline
FPGA lack documentation&0.3&0.5&0.15\\ \hline
Delivered documentation not adequate for usage and upgrade.&0.5&0.3&0.15\\ \hline
Selected software methodology creates issues with team member that decreases productivity&0.2&0.7&0.14\\ \hline
Legal changes significantly affect project - drones in particular&0.2&0.7&0.14\\ \hline
The final product cannot be maintained or extended.&0.7&0.2&0.14\\ \hline

> Client becoming hostile&0.2&0.7&0.14\\ \hline

Reviewer: ```suggestion
Client is not cooperative or does not provide necessary information&0.2&0.7&0.14\\ \hline
```

Client wants to modify scope and requirements that leads to delays or cut features.&0.2&0.6&0.12\\ \hline
Development environment is inadequate.&0.2&0.6&0.12\\ \hline
Sabotage&0.1&1&0.1\\ \hline
Loss of client&0.1&1&0.1\\ \hline
Loss of team member&0.1&0.9&0.09\\ \hline
Equipment shipment is delayed or lost.&0.1&0.8&0.08\\ \hline
Client is not available enough to provide significant help.&0.1&0.5&0.05\\ \hline
Competitive offerings affect project requirement&0.1&0.5&0.05\\ \hline
Camera doesn’t work with FPGA&0.1&0.5&0.05\\ \hline
FPGA doesn’t work with transmission systems&0.1&0.5&0.05\\ \hline

> Ardell becomes hostile&0&0&0\\ \hline

Reviewer: ```suggestion
```
